control/control.openSUSE.xml

```diff
@@ -526,7 +526,7 @@ Firefox as browser, and Nautilus as file manager.
           <label>Server (Text Mode)</label>
         </server>
         <server_description>
-          <label>Server is usefull when computer should not have interaction with user and have only command line.</label>
+          <label>The "Server" role is useful when the computer should not have any interaction with the user and only provide a command line.</label>
```

Comment:
- Please wrap the line.
- The command line *is* a form of interaction! Well, that's another problem of the original description. So
> should not have any *graphical* interaction